sdcm/results_analyze.py

```diff
@@ -681,7 +681,7 @@ def check_regression(self, test_id, is_gce=False):
 
         return True
 
-    def check_regression_with_subtest_baseline(self, test_id, base_test_id, subtest_baseline, is_gce=False):
+    def check_regression_with_subtest_baseline(self, test_id, base_test_id, subtest_baseline, base_test_name=None, is_gce=False):  # pylint: disable=too-many-arguments
```

Comment:
what's this `base_test_name` for ? why to mess with that ?

Reply:
it was needed to correctly filter the cdc docs in ES. Refactored test to use same method as on master.